modules/keymaker/pages/index.adoc

```diff
@@ -0,0 +1,62 @@
+= Keymaker - Recommendations Framework and Cypher Query Pipeline
+:slug: keymaker
+:author: Snehal Maid
+:category: labs
+:tags: graphql, api, cypher, webapp-development, cypher-query-pipelines
+:neo4j-versions: 4.4, 5.x
+:page-pagination:
+:page-product: keymaker
+
+image::keymaker_architecture.png[width=800]
+
+Keymaker is an innovative recommendations (next best action) framework designed to accelerate the development of flexible Neo4j based applications. It is a data model agnostic tool, It seamlessly integrates with Neo4j databases (versions 4.5.x and 5.x supported) allowing for the flexible configuration of analytical engines tailored to specific business needs across diverse industries. With Keymaker, you automatically gain access to a GraphQL API endpoint for each engine you develop, simplifying the process of constructing user interfaces with popular frameworks like React or System to System Interfaces.
+
+== Installation
+
+* Check out the code from the https://github.com/neo4j-labs/keymaker[GitHub Repository]
+* Ensure that you have https://nodejs.org/[Node.js] on your machine
+* Follow the instructions in the Keymaker project README
+
+== Functionality Includes
+
+* *Seamless Integration with Neo4j Databases*. Supports integration with existing Neo4j databases, compatible with versions 4.5.x and 5.x.
+* *Score-Based Recommendation Engine*. Delivers personalized recommendations to users, leveraging scores generated based on predefined business logic.
```

Comment:
bit more detail?

Reply:
Modified the blog to deep dive into the benefits of keymaker. The blog has a sample query pipeline and explains the response of advantages of keymaker.